code/modules/mob/living/simple_animal/friendly/friendly_emote.dm

```diff
@@ -91,3 +91,12 @@
 	key = "purr"
 	key_third_person = "purrs"
 	message = "purrs."
+
+/datum/emote/living/sit/cat
+	message = null
```

Comment:
Also, you should probably set a key (and key_third_person) for good measure, so people can emote *sit or *sits.

Reply:
Been made aware that there already is a `/datum/emote/living/sit`, and so this inheritance pulls from that. Oops.